src/functions.php

```diff
@@ -164,3 +164,54 @@ function getVersion()
 {
     return 'v2.6.0';
 }
+
+/**
+ * @param $array
+ * @param $attribute
+ * @param $currentValue
+ * @return false|float
+ */
+function gradientAlphaFor($array, $attribute, $currentValue) {
+
+    // memory cache
+    static $caches;
+    if(null == $caches) {
+        $caches = [];
+    }
+
+    if(!isset($caches[$attribute])) {
+        // avoid to iterate over array too many times
+        $max = 0;
+        $min = 1;
+        foreach($array as $item) {
+            if(!isset($item[$attribute])) {
+                continue;
+            }
+
+            $max = max($max, $item[$attribute]);
+            $min = min($min, $item[$attribute]);
+        }
+
+        $caches[$attribute]['max'] = $max;
+        $caches[$attribute]['min'] = $min;
+    }
+
+    $max = $caches[$attribute]['max'];
+    $min = $caches[$attribute]['min'];
+
+
+    $percent = (($currentValue - $min) * 100) / ($max - $min);
```

Comment:
You should manage the case that min and max are equals, leading to division by 0